### PR TITLE
Update configure-exchange-server-for-hybrid-modern-authentication.md

### PR DESCRIPTION
**Pull request by mukeshHCW**
added more clarity in Note for Domain Name parameter

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md [hunk 1]
```diff
@@ -172,6 +172,8 @@ Set-OrganizationConfig -OAuth2ClientProfileEnabled $true
 
 > [!NOTE]
 > In case EXCH is in hybrid with **multiple tenants**, there are multiple AuthServer objects present in EXCH with domains corresponding to each tenant.  The **IsDefaultAuthorizationEndpoint** flag should be set to true (using the **IsDefaultAuthorizationEndpoint** cmdlet) for any one of these AuthServer objects. This flag can't be set to true for all the Authserver objects and HMA would be enabled even if one of these AuthServer object's **IsDefaultAuthorizationEndpoint** flag is set to true.
+> 
+> For **DomainName** parameter, use Tenant Domain value which is usually of the form contoso.onmicrosoft.com
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md]:
```suggestion
> For the **DomainName** parameter, use the tenant domain value, which is usually in the form **contoso.onmicrosoft.com**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mukeshHCW** [comment]
approved copy review suggestions.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**